qualification/recv.py

```diff
@@ -457,19 +466,20 @@ def _create_receive_stream_group(
 
         if use_ibv:
             config = spead2.recv.UdpIbvConfig(
-                endpoints=endpoints, interface_address=interface_address, buffer_size=64 * 1024 * 1024, comp_vector=core
+                endpoints=endpoints,
+                interface_address=interface_address,
+                buffer_size=128 * 1024 * 1024,
```

Comment:
You mentioned in JIRA IIRC about increasing this to 10s of GB.

Would that be here? Or perhaps just allocating more 128MB buffers to the ring?

Reply:
That's another potential change, which I have up my sleeve if needed. I'm hoping I don't need that much memory. That needs to be done in the data_ringbuffer size calculation rather than here, because ibverbs doesn't support arbitrarily large buffers.